packages/openapi-parser/src/types.ts

```diff
@@ -15,6 +16,11 @@ export interface OpenAPICustomSpecProperties {
      * This option can be used to hide code samples for the entire spec.
      */
     'x-hideTryItPanel'?: boolean;
+
+    /**
+     * Description in HTML format.
```

Comment:
This one is purely internal, right?

Should we expose it?

Reply:
Let's at least comment its use

Reply:
We don't really expose it as the others.